changelogs/changelog.yaml

```diff
@@ -99,3 +99,34 @@ releases:
     fragments:
     - 51-doc-plugin-fixes.yaml
     release_date: '2020-06-23'
+  1.0.2:
+    changes:
+      bugfixes:
+      - Added workaround to avoid set_fact dynamically assigning value. This behavior
+        seems to have been broken after ansible2.9.
+      - Make `src`, `backup` and `backup_options` in vyos_config work when module
+        alias is used (https://github.com/ansible-collections/vyos.vyos/pull/67).
+      - vyos_config - fixed issue where config could be saved while in check mode
+        (https://github.com/ansible-collections/vyos.vyos/pull/53)
+      minor_changes:
+      - Add changelogs for previous release (https://github.com/ansible-collections/vyos.vyos/pull/52)
+      - Fixed the typo in the modulename of ospfv2 and ospfv3 unit tests.
+      - Updated docs.
+      - terminal plugin - Added additional escape sequence to be removed from terminal
+        output.
+    fragments:
+    - 52-add-change-log-1.0.0.yaml
```

Comment:
It doesn't matter a whole bunch, but this fragment doesn't technically belong here as we've removed it from the changelog

```suggestion
```